### ISSUE TITLE
Special characters being rendered in Unicode format

### ISSUE THREAD

**Issue by sc250024** [norwoodj/helm-docs#24]
# Description

You can file this one under "Is this a feature or a bug?" Basically, when a special character like `&` is present in an entry in `values.yaml`, the rendered `README.md` file shows the character as a Unicode sequence.

I encountered this because the [elastic/elasticsearch](https://hub.kubeapps.com/charts/elastic/elasticsearch) charts allows specifying a value for `clusterHealthCheckParams`, which I override in the following way:

```yaml
elasticsearch:
  # elasticsearch.clusterHealthCheckParams -- The Elasticsearch cluster health status params that will be used by readinessProbe command
  clusterHealthCheckParams: "wait_for_status=yellow&timeout=1s"
```

This results in the following output `README.md` file:

```markdown
## Chart Values

| Key | Type | Default | Description |
|-----|------|---------|-------------|
| elasticsearch.clusterHealthCheckParams | string | `"wait_for_status=yellow\u0026timeout=1s"` | The Elasticsearch cluster health status params that will be used by readinessProbe command |
```

# Expected result

Special characters such as `&` should *not* be rendered as a Unicode sequence, and should show up as normal in the `README.md` file as well.

# Current result

A character like `&` will get rendered as `\u0026`.

**Comment by norwoodj** [comment]
Hey @sc250024. Thanks for bringing this to my attention. Suspect this is an issue with the gotemplate renderer. Will definitely take a look at fi

**Comment by norwoodj** [comment]
Hey @sc250024! Sorry for taking so long with this. I have now fixed the issue. It will be in v0.11.0 which I will release when I fix these last couple of open issues.

**Comment by norwoodj** [comment]
This has been released in [v0.11.0](https://github.com/norwoodj/helm-docs/releases/tag/v0.11.0)

**Comment by sc250024** [comment]
A scholar and a gentleman you are! Thank you sir!
